packages/react/src/components/DocumentCard/DocumentCardActions.base.tsx

```diff
@@ -44,8 +44,10 @@ export class DocumentCardActionsBase extends React.Component<IDocumentCardAction
 
         {views! > 0 && (
           <div className={this._classNames.views}>
-            <Icon iconName="View" className={this._classNames.viewsIcon} />
-            {views}
+            <>
```

Comment:
This perplexed me a bit as the parent div only wanted a single child when views was a child